### DISCUSSION TITLE
How is MACS2 significance normalized for sequencing depth? And where does it occur in addReproduciblePeakSet function?

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#1411] [Questions / Documentation]
Hi @rcorces,

I am reading up on the iterative overlap procedure in preparation for my upcoming PhD oral exam :)

From the documentation: 
"First, ArchR would call peaks for each pseudo-bulk replicate individually. Then, ArchR would analyze all of the pseudo-bulk replicates from a single cell type together, performing the first iteration of iterative overlap removal. **It is important to note that ArchR uses a normalized metric of significance for peaks to compare the significance of peaks called across different samples. This is because the reported MACS2 significance is proportional to the sequencing depth so peak significance is not immediately comparable across samples.** After the first iteration of iterative overlap removal, ArchR checks to see the reproducibility of each peak across pseudo-bulk replicates and only keeps peaks that pass a threshold indicated by the reproducibility parameter. At the end of this process, we would have a single merged peak set for each of the 3 cell types, A, B, and C."

Could you help me locate which line in this script (`addReproduciblePeakSet`) does the normalization of MACS2 p-values to correct for sequencing depth?
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/ReproduciblePeakSet.R

Thank you!

**Comment by rcorces** [comment]
I believe this is handled via quantile normalization and reflected in multiple calls to `nonOverlappingGR()` using `replicateScoreQuantile` or `groupScoreQuantile`.
